### PR TITLE
Recipe:app

### PR DESCRIPTION
**Pull request by Shubh-Dev**
UPDATE!
Readme and License updated as per the respected reviewer's suggestion. 

UPDATE!

- Food page is the root path now
- After sign-up user_url error fixed

UPDATE!
screenshot of the permission main added.
Hello Dear reviewer,
Because of my **partner's unavailability,** I am working solo on this group project.
I implemented almost everything as per the requirements.
Let me know if everything is alright. 
Permission:
permission_main

### CODE REVIEW

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @Shubh-Dev  👋 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️ 

- Sorry that your partner was not available to do the capstone with you. Unfortunately, I cannot continue to review your pull request because as a requirement, if your partner is unavailable, you are required to attach a screenshot to your pull request from student success permitting you to work on this capstone project solo.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: config/routes.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   root to: 'recipes#index'
```
Comment (DammyShittu) [inline config/routes.rb]: - [ ] The food index page should be the root path. Kindly fix that.

File: app/views/devise/registrations/new.html.erb [hunk 2]
```diff
@@ -1,34 +1,36 @@
...
   <div class="actions">
-    <%= f.submit "Sign up" %>
+    <%= f.submit "Sign up", class:"classic-btn" %>
   </div>
 <% end %>
-
+<br>
 <%= render "devise/shared/links" %>
+
+</div>
```
Comment (DammyShittu) [inline app/views/devise/registrations/new.html.erb:36]: - [ ] I get this error page when I sign up as a user. Kindly rectify it. The user is actually created but it shows this unnecessary error.

  nfjf

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Shubh-Dev,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by FrederickMih — COMMENTED** [review]
Hi @Shubh-Dev, 

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

The README.md file for this project does not have the Acknowledgment header. Also, make sure the License section has the header of [this](https://github.com/microverseinc/readme-template/edit/master/README.md) template.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @Shubh-Dev  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**